README-YAML.md

```diff
@@ -211,3 +212,111 @@ The following tables describe the format in more detail.
+
+Just like for loop invariants, the verification entry consists of four parts:
+the entry type,
+the metadata to describe the provenance of the entry,
+the location that this entry talks about, and
+the invariant at that location.
+
+```yaml
+- entry_type: location_invariant
+  metadata:
+    format_version: 0.2
+    uuid: 92c380d6-00a1-4e97-8c63-0f246206c6ab
+    creation_time: 2022-05-16T11:56:13.480768Z
+    producer:
+      name: A2Y
+      version: 1.0
+    task:
+      input_files:
+        - c/loops/trex04.c
+      input_file_hashes:
+        c/loops/trex04.c: f70ffe9cd45c37f44e9e780e31340fab45b6a2fb7f7ef23a2d90faf4241229d6
+      specification: CHECK( init(main()), LTL(G ! call(reach_error())) )
+      data_model: ILP32
+      language: C
+  location:
+    file_name: c/loops/trex04.c
+    file_hash: f70ffe9cd45c37f44e9e780e31340fab45b6a2fb7f7ef23a2d90faf4241229d6
+    line: 47
+    column: 0
+    function: main
+  location_invariant:
+    string: x <= 0
+    type: assertion
+    format: C
+```
+
+#### Description
+
+The following tables describe the format in more detail.
+
+##### entry
+| Property              | Data Type     | Format                            | Description   |
+|---                    | ---           | ---                               | ---           |
+| `entry_type`          | string        | "location_invariant"              | The type of this entry. The format is specific to the entry type. In this case, the entry type declares that the entry contains a location invariant that holds at some location in a program. |
+| `metadata`            | assoc. array  | [see below](#metadata-1)          | Additional information about the "environment" in which the entry was produced. |
+| `location`            | assoc. array  | [see below](#location-1)          | Location in the source code to which the entry refers, i.e., at which the invariant holds. |
+| `location_invariant`  | assoc. array  | [see below](#location_invariant)  | Actual location invariant. |
+
+##### metadata
+| Property          | Data Type     | Format                    | Description  |
+|---                | ---           | ---                       | ---          |
+| `format_version`  | string        | Format version specifier  | Version of the verification-entries format that the entry is formatted in. The entry type "location_invariant" was introduced in version 0.2 of the format.|
+| `uuid`            | string        | UUID                      | Unique identifier of the entry ([RFC4122](https://www.ietf.org/rfc/rfc4122.txt) defines the UUID format).  |
+| `creation_time`   | string        | ISO 8601                  | Date and time when the entry (not the file) was created. |
+| `producer`        | assoc. array  | [see below](#producer-1)  | Tool that produced the entry. |
+| `task`            | assoc. array  | [see below](#task-1)      | Verification task during which the entry was produced. |
+
+##### producer
```

Comment:
It might get a bit redundant if we start copying sections like this. I would rather have the description refactored such that we have a section for common keys like `producer` and then just point to that section for the different entry types.

Reply:
Here's a radical idea: ditch the hand-written Markdown altogether and have everything in a nice JSON schema, see #61.